os/drivers/lcd/mipi_lcd.c

```diff
@@ -399,6 +405,38 @@ static int lcd_setcontrast(FAR struct lcd_dev_s *dev, unsigned int contrast)
 	return OK;
 }
 
+FAR void lcd_set_logo(FAR struct lcd_dev_s *dev)
```

Comment:
How about lcd_put_logo or lcd_flush_logo or lcd_show_logo or something like that?
`set` looks just setting some value and looks flushing will be called later.

Reply:
fixed.